www_src/lib/router.jsx

```diff
@@ -30,9 +30,22 @@ module.exports = {
 
     return r;
   },
+  getRouteData: function () {
+    var r = {};
```

Comment:
nit: minification will take care of `r`, let's call it `routes` or something in the full code?